CS3219-AY2425S1/cs3219-ay2425s1-project-g11
# Feat/question service

This PR aims to add create, update and delete operations to the `question-service`

### How to test
- Go to the postman docs
- Make sure that `POST localhost:4001/questions/` and `PUT localhost:4001/questions/:questionid` is able to take in a request body like
```
{
  "difficulty": [1..3],
  "description": string,
  "examples": string[],
  "constraints": string,
  "tags": string[],
  "title_slug": string,
  "title": string,
  "pictures": any[]
}
```
- Make sure that `DELETE localhost:4001/questions/:questionid` works and return the correct id
